Buildings/Resources/Scripts/BuildingsPy/conf.yml

```diff
@@ -171,12 +171,14 @@
   optimica:
     comment: timeout on travis after 300s, https://app.travis-ci.com/github/lbl-srg/modelica-buildings/jobs/618700275
     time_out: 600
+- model_name: Buildings.Fluid.CHPs.OrganicRankine.Validation.VariableSource
+  openmodelica:
+    comment: 'division by zero at time 0 and 0.6, (a=0) / (b=0), where divisor b expression is: orc.m1_flow * orc.cyc.cpHot'
+    simulate: false
 - model_name: Buildings.Fluid.Examples.FlowSystem.Basic
   openmodelica:
     comment: '''omc'' caused ''simulation terminated by an assertion at initialization''.'
     simulate: false
-  optimica:
```

Comment:
Not sure why this shows up in this PR as the ORC model does not affect `FlowSystem.Basic`

Reply:
Fixed in 6784e30.
I must have deleted these lines by mistake when trying to fix the file after I didn't put in the model in alphabetical order.